discovery_engine/test/integration/init_engine_test.dart

```diff
@@ -40,23 +41,17 @@ void main() {
     });
 
     test('init engine with ai models', () async {
-      server = await LocalNewsApiServer.start();
-      final engine = await DiscoveryEngine.init(
-        configuration: createConfig(data, server.port),
-      );
-
+      final engine = await initEngine(data, server.port);
       expect(engine, isA<DiscoveryEngine>());
+      await engine.dispose();
```

Comment:
this line could be moved to the `tearDown()` like in the other tests. similarly below.